Rlog/View/Schedule/ScheduleListView.swift

```diff
@@ -2,13 +2,210 @@
 //  ScheduleListView.swift
 //  Rlog
 //
-//  Created by Kim Insub on 2022/11/10.
+//  Created by Noah's Ark on 2022/11/08.
 //
 
 import SwiftUI
 
 struct ScheduleListView: View {
+    @ObservedObject var viewModel = ScheduleListViewModel()
+    @State var selection = 1
+    var currentMonth: String {
+        let components = Calendar.current.dateComponents([.year, .month], from: viewModel.currentDate)
+        let year = components.year ?? 2000
+        let month = components.month ?? 1
+        
+        return "\(year). \(month)"
+    }
+    var currentWeek: [CalendarModel] {
+        return viewModel.getWeekOfDate(viewModel.currentDate)
+    }
+    var previousWeek: [CalendarModel] {
+        return viewModel.getWeekOfDate(viewModel.previousDate)
+    }
+    var nextWeek: [CalendarModel] {
+        return viewModel.getWeekOfDate(viewModel.nextDate)
+    }
+    let mockData: [WorkspaceEntitySample] = [
+        WorkspaceEntitySample(
+            name: "팍이네 팍팍 감자탕",
+            schedules: ScheduleEntitySample(),
+            workdays: WorkdayEntitySample(date: Date(), endHour: 18)
+        ),
+        WorkspaceEntitySample(
+            name: "팍이네 팍팍 감자탕",
+            schedules: ScheduleEntitySample(),
+            workdays: WorkdayEntitySample(date: Date(), endHour: 15)
+        ),
+        WorkspaceEntitySample(
+            name: "팍이네 팍팍 감자탕",
+            schedules: ScheduleEntitySample(),
+            workdays: WorkdayEntitySample(date: Date(), endHour: 22)
+        )
+    ]
+
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        VStack {
+            header
+            scheduleContainer
+        }
+        .background(.gray)
+    }
+}
+
+private extension ScheduleListView {
+    var header: some View {
+        HStack(spacing: 0) {
+            Group {
+                Button {
+                } label: {
+                    Image(systemName: "chevron.left")
+                }
+                Text(currentMonth)
+                Button {
+                } label: {
+                    Image(systemName: "chevron.right")
+                }
+            }
+            .font(.title)
+            .foregroundColor(.black)
```

Comment:
이 부분도 제 PR(https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Bremen-DD/pull/74#discussion_r1020779551)에 남겨주신 리뷰처럼 컴포넌트로 수정될 예정인가요?

Reply:
+ 폰트용 블랙이 따로 있어서 적용해주세면 좋을 것 같습니다

```suggestion
        HStack(spacing: 0) {
            Group {
                Button {
                } label: {
                    Image(systemName: "chevron.left")
                }
                Text(currentMonth)
                Button {
                } label: {
                    Image(systemName: "chevron.right")
                }
            }
            .font(.title)
            .foregroundColor(Color.fontBlack)
```

Reply:
컴포넌트 작업 진행 예정입니다.
[93b187f](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Bremen-DD/pull/72/commits/93b187f9faed7c7e88f296c01463e4c5091c322d) 반영했습니다.